frontend/src/pages/chat/components/chat-layout/chat-layout.tsx

```diff
@@ -86,9 +92,17 @@ const ChatLayout: React.FC<Properties> = ({ filter }) => {
               />
             );
           })}
+        {isChatbotReplyLoading && (
+          <div>
+            <span className={styles['loader']} />
```

Comment:
why can't we use our loader component?

Reply:
we can't, because it has a position absolute so it would overlap the whole page. To use our loader component I would need to pass additional props and change some stylings

Reply:
Let's pass additional props to the existing loader, something like - isOverflow, and we can also wrap it in the div in loader component using conditional rendering

Reply:
sure. Then I will extend the component which we have and add this prop to existing ones

Reply:
Yeah, the same approach is needed in meditation modal while loading

Reply:
I updated all occurrences of Loader